### ISSUE TITLE
Controls missing after update to 4.6.2

### ISSUE THREAD

**Issue by mforkel** [CDrummond/lms-material#899]
After the update to version 4.6.2 on Lyrion Music Server 8.4.0, the controls for start/stop and skip are not displayed anymore on an iPad with iPadOS 15.8.2:

![grafik](https://github.com/CDrummond/lms-material/assets/1212429/e4174f58-e3c6-47ef-916c-e3c3b3c778d6)

The problem does not occur with Firefox on Windows 10.

**Comment by CDrummond** [comment]
Does this only affect this bottom now-playing bar? What about if you expand now-playing? Also, only desktop layout or does it affect mobile layout too? iPhone as well?

Sorry for all the questions - but I'm not an iOS user.

**Comment by CDrummond** [comment]
I've asked on forums.slimdevices.com and no one else has this issue. Perhaps its a caching issue? Either way, without more info there is nothing I can do.

**Comment by mforkel** [comment]
Sorry for the late reply. 

Today I noticed that on a second iPad with the same software versions everything looks fine. So I went back to the first iPad - still the same problem after a page update. Then I opened a another tab - no problems there. So finally I switched back to the first tab, and after another update the problems was gone. 

I'm sorry for the false alarm. It definitely was a local problem, but I can't say what really went wrong.
